Dockerfile

```diff
@@ -126,8 +126,8 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     tk-dev \
     && sudo rm -rf /var/lib/apt/lists/*
 
-ARG PYTHON_VERSION=3.10.8
-ARG PYENV_GIT_TAG=v2.3.7
+ARG PYTHON_VERSION=3.11.4
```

Comment:
> to fix https://github.com/oss-review-toolkit/ort/issues/7333

It's confusing to refer to that issue as it's already closed.

Reply:
Can you explain why upgrading python is needed in order to support 3.11?

In particular, would it suffice to stick to 3.10 (not saying we should) and only pass `-p 3.11` to python inspector by default ?